doc/data/messages/c/consider-using-namedtuple-or-dataclass/good.py

```diff
@@ -1 +1,8 @@
-# This is a placeholder for correct code for this message.
+from collections import namedtuple
+
```

Comment:
We should use the new style (creating a class), this one is lower performance because there's an exec.

Reply:
What is the new style? Also, I got the information for namedtuples that I used in the code from this URL: https://realpython.com/python-namedtuple/

Reply:
```python
from typing import NamedTuple

class MAPPING_1(NamedTuple):
    key_1: str
    key_2: str
    key_diff_1: str
    key_diff_2: str
```

Reply:
Changed